### ISSUE TITLE
tensorboard could not found event files if path has Chinese charactors under Windows.

### ISSUE THREAD

**Issue by jumpapex** [tensorflow/tensorboard#861]
Dear team,

On my machine, tensorboard inspect works, but no graph shown in web(it says no event files found) under Windows.

Environment:
tensorflow 1.4.0
tensorboard 0.4.0r3 
Windows 7 with sp164bit
Python 3.5.2 64bit

**Run tensorboard as:**
D:\PythonProjects\MNIST入门>tensorboard --logdir=log2
web page shows "no event files found"

**But inpsect works very will:**
D:\PythonProjects\MNIST入门>tensorboard --logdir=log2 --inspect
  Found event files in:
  log2\test
  log2\train

  These tags are in log2\test:
  audio - 
  histograms

**If I copy the log2 directory to D:\PythonProjects\log**
run D:\PythonProjects>tensorboard --logdir=log, everyting is OK.

To trace the source code:
I found:
for inspect command, it enumerates the directory with the raw logdir argument(e.g. log)
for web, in testboard/backend/application.py
    def parse_event_files_spec
         ...
         if uri_pattern.match(path) is None:
             path = os.path.realpath(path)        # path convert to full path
    
after that, when enumerate the directory, is_directory(file_io.py) always return false.
If I comment the line of path convert, everything is OK.
Please take a look of is_directory(indee pywrap_tensorflow.IsDirectory).

Regards and Thanks!

Jumpapex.

**Comment by nfelt** [comment]
This looks like it might be a duplicate of #682 which we closed since we didn't hear back enough from the OP to isolate the issue.  Could you try running this from a console in the directory **containing** the directory with chinese characters in its name, i.e. the `D:\PythonProjects` directory?

```
python -c "import os; import tensorflow as tf; status = tf.pywrap_tensorflow.TF_NewStatus(); print(tf.pywrap_tensorflow.GetChildren(tf.compat.as_bytes(os.getcwd()), status))"
```

**Comment by nfelt** [comment]
@jumpapex Sorry for the long wait on this.  This seems to be an underlying issue in which tf.gfile functions don't really understand Windows filenames that have Unicode characters, so I've filed https://github.com/tensorflow/tensorflow/issues/17695 against TensorFlow for that - you may want to subscribe to that issue.

If that analysis is correct, this isn't something we can really resolve properly in TensorBoard since we rely on tf.gfile for file operations, so for now I'm going to close this issue.  However, if we establish that TensorBoard is actually doing something incorrect in terms of using tf.gfile, then we can re-open this issue.
